src/book.js

```diff
@@ -128,16 +128,6 @@ export class Book {
         });
       } else {
         if (!this.cropbox) {
-          const cropBox = page.getCropBox();
-          const bleedBox = page.getBleedBox();
-          const trimBox = page.getTrimBox();
-          const artBox = page.getArtBox();
-          console.log(
-            `\n\tCropBox [${cropBox}]`,
-            `\n\tBleedBox [${bleedBox}]`,
-            `\n\tTrimBox [${trimBox}]`,
-            `\n\tArtBox [${artBox}]`
-          );
```

Comment:
Unfortunately this console logs are actually a bit expensive. We could comment them out or add a debug flag somewhere for verbose output, but we shouldn't be shipping this.

Reply:
👍 I can't say I'm aware ATM of any logs I value... I think I tend to add them when I need them and shortly after but then... the problem's always somewhere else next time...